src/e2e-test/java/io/cdap/plugin/bigquerymultitable/stepdesign/BQMultiTableValidation.java

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright © 2023 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.bigquerymultitable.stepdesign;
+
+import com.google.cloud.bigquery.FieldValue;
+import com.google.cloud.bigquery.FieldValueList;
+import com.google.cloud.bigquery.TableResult;
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import io.cdap.e2e.utils.BigQueryClient;
+import io.cdap.e2e.utils.PluginPropertyUtils;
+import org.junit.Assert;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.Types;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+
+/**
+ * BQMTMultiTable Validation.
+ */
+
+public class BQMultiTableValidation {
+
+  private static final String tabA = "tabA";
+
+  private static final String tabB = "tabB";
+  public static Connection getMysqlConnection() throws SQLException, ClassNotFoundException {
+    String database = PluginPropertyUtils.pluginProp("databaseName");
+    Class.forName("com.mysql.cj.jdbc.Driver");
+    Connection connection = DriverManager.getConnection("jdbc:mysql://" + System.getenv("MYSQL_HOST") + ":" +
+                                     System.getenv("MYSQL_PORT") + "/" + database + "?tinyInt1isBit=false",
+                                     System.getenv("MYSQL_USERNAME"), System.getenv("MYSQL_PASSWORD"));
+    return connection;
+  }
+
+  /**
+   * Validates data from a MySQL table against corresponding BigQuery tables.
+   *
+   * @param sourceTable The name of the MySQL table to be validated.
+   * @return true if validation passes for all tables, false otherwise.
+   * @throws IOException If an I/O error occurs.
+   * @throws InterruptedException If the current thread is interrupted while waiting.
+   * @throws SQLException If an SQL error occurs.
+   * @throws ClassNotFoundException If the class specified is not found.
+   */
+
+  public static boolean validateMySqlToBQRecordValues(String sourceTable) throws IOException, InterruptedException,
```

Comment:
So we are using Multiple Database table plugin here but we are using a SQL Database/Table. It would be good if we can add few comments and context here describing this in this class.